### PR TITLE
A basic version 1.0 of the component library

### PR DESCRIPTION
**Pull request by umbobabo**
Please review all the code.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -113,19 +118,22 @@
...
     "@economist/component-imagecaption": "^1.3.0",
     "@economist/component-masthead": "^2.1.3",
     "@economist/component-palette": "^1.0.0",
+    "@economist/component-typography": "^1.1.1",
     "react": "^0.13.3",
     "react-simpletabs": "^0.6.1"
   },
   "devDependencies": {
-    "@economist/component-devpack": "^3.0.0",
+    "@economist/component-devpack": "latest",
```
Comment (fabiosantoscode) [inline package.json]: This here is dangerous. component-devpack is pretty unstable

**Comment by sublimino** [comment]
LGTM after the `component-devpack` change

**Comment by fabiosantoscode** [comment]
What he said.

About the fonts, I have the changeset right here but can only push after we merge yours.

**Comment by sublimino** [comment]
Merged in good faith and sound mind.

`latest` to be updated afterwards.

**Pull request merged**